geniza/corpus/templates/corpus/document_transcribe.html

```diff
@@ -27,6 +27,7 @@
     <!-- document details -->
     <h1 class="sr-only">{{ page_title }}</h1>
     {% include "corpus/snippets/document_header.html" %}
+    <h3>{{ source_detail }}</h3>
```

Comment:
can we make this not bold / not h3 ? and use the formatted version of the citation that we display at the top of the transcription panel. (I guess we have to add notes separately?)